cmd/flowserver/expression.go

```diff
@@ -39,11 +38,7 @@ func handleExpression(w http.ResponseWriter, r *http.Request) (interface{}, erro
 		return nil, fmt.Errorf("missing context or expression element")
 	}
 
-	// build up our context
-	context, err := runs.ReadContext(expression.Context)
```

Comment:
I'm not sure what the plans are for this enpoint but it's not to easy to unmarshall a run or a contact these days without other information, e.g. channels - so I'm switching this to take a generic bod as context.